plugins/coreboot/fu-coreboot-sysfs.c

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (C) 2019 9elements Agency GmbH <patrick.rudolph@9elements.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <string.h>
+#include <stdio.h>
+
+#include "fu-plugin-coreboot.h"
+
+#define SYSFS_BUS "/sys/bus/coreboot/"
```

Comment:
This should go into fu-common as a directory to lookup, especially if it's used for unit tests down the road.